pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -908,6 +908,35 @@ private void charge4MenuItem_Click(object sender, EventArgs e)
             _graphSpectrumSettings.ShowCharge4 = !_graphSpectrumSettings.ShowCharge4;
         }
 
+        private void SynchMzScaleToolStripMenuItemClick(object sender)
+        {
+            if (_graphFullScan == null || !_graphFullScan.Visible
+                                       || _graphSpectrum == null || !_graphSpectrum.Visible)
+                return;
+            Settings.Default.SyncMZScale = synchMzScaleToolStripMenuItem.Checked;
+            //testing support
+            var source = sender as IMzScaleCopyable ??

```

Comment:
The "//testing support" comment only applie to the "as IMzScaleCopyable" part of this statement.
You should probably make this into a full "if ... else" statement so that you can comment the two parts of the if statement separately.

Reply:
This was an if statement originally but has been changed to inline ?? by Brendan's request. Could you guys decide between yourself which one is better and let me know your verdict.

Reply:
This was the original Brendan's comment:
> Assigning senter to source seems like it could be done using ?? followed by your other expression.